src/Resources/model/Machine.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, lab, name):
         self.bridge = None
 
         self.meta = {}
+        self.sysctls = {}
```

Comment:
I would like to have the sysctls inside the `meta` dict instead of another dict.

Reply:
@Skazzino This wouldn't allow for multiple sysctls, unless we add a prefix

Reply:
Yes, I was thinking to have a subdict. Something like:
```python
self.meta['sysctls']['net......'] = 1
```

Reply:
Alright I'll fix that